testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCreateTableCompatibility.java

```diff
@@ -310,11 +312,19 @@ public void testCreateTableWithColumnCommentOnDelta()
         }
     }
 
-    private String getDatabricks104DefaultTableProperties()
+    private String getDatabricksDefaultTableProperties()
     {
-        return "TBLPROPERTIES (\n" +
-                "  'Type' = 'EXTERNAL',\n" +
```

Comment:
Did 11.3 get rid of the type param?

Reply:
Yes from `SHOW CREATE TABLE` result. `DESCRIBE EXTENDED` still provides such information.